### PR TITLE
[#9382] Add numerical question creation example box

### PR DESCRIPTION
**Pull request by Derek-Hardy**
Part of #9382 

num-scale

### CODE REVIEW

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts [hunk 1]
```diff
@@ -49,6 +52,36 @@ export class InstructorHelpQuestionsSectionComponent extends InstructorHelpSecti
...
+
+    giverType: FeedbackParticipantType.STUDENTS,
+    recipientType: FeedbackParticipantType.OWN_TEAM_MEMBERS,
+
+    numberOfEntitiesToGiveFeedbackToSetting: NumberOfEntitiesToGiveFeedbackToSetting.CUSTOM,
+    customNumberOfEntitiesToGiveFeedbackTo: 1,
+
+    showResponsesTo: [],
+    showGiverNameTo: [],
+    showRecipientNameTo: [],
```
Comment (madanalogy) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts:75]: Would be good to follow the defaults created for a new question https://github.com/TEAMMATES/teammates/blob/331faaa71954c7530125aca3fefebca6d27c2837/src/web/services/feedback-questions.service.ts#L332-L348

Comment (xpdavid) [reply src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts:75]: I think it is even better to use the service to generate a template and them fill dummy values such as `questionBrief` or `questionDescription`.

Comment (madanalogy) [reply src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts:75]: > I think it is even better to use the service to generate a template and them fill dummy values such as `questionBrief` or `questionDescription`.

You mean create a new method in the service specifically for populating help page data? I like the idea but feel that it's something we can work on after v7?

Comment (xpdavid) [reply src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts:75]: It's fine. I am thinking about something else. Just ignore my comment :P

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts [hunk 2]
```diff
@@ -51,6 +54,29 @@ export class InstructorHelpQuestionsSectionComponent extends InstructorHelpSecti
     showRecipientNameTo: [FeedbackVisibilityType.INSTRUCTORS, FeedbackVisibilityType.RECIPIENT],
   };
 
+  readonly exampleNumericalScaleEditFormModel: QuestionEditFormModel = {
```
Comment (xpdavid) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts:57]: Just a nit. The order of fields can be more logically (even it does not matter in JavaScript).

- Like the `customNumberOfEntitiesToGiveFeedbackTo` should be together with `numberOfEntitiesToGiveFeedbackToSetting`. 
- Form control fields (`isEditable`, `isSaving`) can be separated from the actual form fields.

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html [hunk 3]
```diff
@@ -162,7 +162,9 @@ <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes
               Specify the feedback path that should be used to generate the appropriate feedback recipients
             </li>
           </ol>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+              <tm-question-edit-form [(formModel)]="exampleNumericalScaleEditFormModel" [isDisplayOnly]="true"></tm-question-edit-form>
```
Comment (xpdavid) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html:166]: Let's just follow the convention in other PR by not subscribing to the change event (i.e. There is no need to have the `()` (the banana syntax)).

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts (hunk 2 above)
Comment (Derek-Hardy) [reply src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts:60]: Reordered fields for all question model for readability.

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html (hunk 3 above)
Comment (Derek-Hardy) [reply src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html]: Standardised!

**Pull request merged**